README.md

```diff
@@ -2,7 +2,7 @@
 Capstone Design Project - Team 04
 
 ## Project Description
-Through this project, we hope to build a solid and user-friendly frontend interface for a domain modelling application. This app is important as it will allow students registered in software engineering classes to practice and self learn domain modelling in and outside of class times. This will be an improved cross-platform version of TouchCORE.
+Through this project, we hope to build a solid and user-friendly frontend interface for a domain modelling application, which may potentially have a broad impact on students leaning software engineering, especially domain modeling. This app is important as it will allow students registered in software engineering classes to practice and self-learn domain modelling in and outside of class times. This will be an improved cross-platform version of TouchCORE.
```

Comment:
minor change needed. - Through this project .... leaning towards software engineering...